includes/admin/class-wc-payments-admin.php

```diff
@@ -7,6 +7,9 @@
 
 defined( 'ABSPATH' ) || exit;
 
+use Automattic\WooCommerce\Admin\Loader;
+use Automattic\WooCommerce\Admin\Features\Navigation\Menu;
```

Comment:
These imports are no longer used (and they would make the site crash on older versions of WC-Admin I think?). Let's remove them.

Reply:
They would not cause a crash, but agreed they're no longer needed if we go forward with 5717.